app/views/bulkrax/importers/show.html.erb

```diff
@@ -101,6 +101,8 @@
                   <td><%= e.id %></td>
                   <% if e.status == "Complete" %>
                     <td><span class="glyphicon glyphicon-ok" style="color: green;"></span> <%= e.status %></td>
+                  <% elsif e.status == "Pending" %>
```

Comment:
should "Deleted" also be added here somewhere?

Reply:
I left deleted as the `else` category since that is the red `x` I thought it matched well. I can also add a seperate symbol for deleted though if you think that would be better.

<img width="2519" alt="Screen Shot 2021-10-08 at 15 31 48" src="https://user-images.githubusercontent.com/6759631/136631419-48f72347-ade0-47f1-af04-5bff56a09c94.png">

Reply:
I think having it be a green x is probably right or the same as complete.  Red X means failed here, so the delete succeeding should be different I think